packages/js-dash-sdk/src/SDK/Client/Platform/Platform.ts

```
@@ -58,6 +61,9 @@ export interface PlatformOpts {
 interface Records {
   broadcast: Function,
   create: Function,
+  transfer: Function,
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)